### PR TITLE
Vet-Clinic: Feature/query and update

### PR DESCRIPTION
**Pull request by Megagig**
In this PR the following is done:

 Add a new column to the animals' table
 Insert data into animals' table
 Create a transaction
 Update/delete/commit/rollback transaction
 Query table using aggregates
The screenshot of the results is attached to this PR:

Rollback to savpoint
ROLLBACK
update
delete animal

minimum
Average
present
never escaped
Average escape

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,3 +9,100 @@ SELECT * FROM animals WHERE neutered = true;
...
+
+--Rollback the transaction
+ROLLBACK;
+
+--Verify that the transaction was successful
+SELECT * FROM animals;
+
+--Update the species colunm
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
```
Comment (Davidosky007) [inline queries.sql]: * Kindly write queries for these missing transactions as its a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20changes,persist%20after%20commit.)
```
 Verify that changes were made.
Commit the transaction.
Verify that changes persist after the commit.
```

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Megagig 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ SQL commands put in the correct files
✔️ Query results screenshots added to PR description.
✔️ Use of correct Gitflow 👍

## Required Changes ♻️
* Kindly fix the broken MIT link in your readme 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Megagig) [reply queries.sql]: @Davidosky007 
Thanks for reminding me

Comment (Megagig) [reply queries.sql]: This is resolved. Thanks

**Review by Gambit142 — APPROVED** [review]
Hi @Megagig 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Everything that creates or changes the structure of the table is in the schema.sql file.
- Everything that populates tables with data is in the data.sql file.
- Everything that is a query that helps to analyze data is in the queries.sql file.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**